tests/trainer/test_dataloaders.py

```diff
@@ -658,6 +656,62 @@ def test_warning_with_few_workers(mock, tmpdir, ckpt_path):
         trainer.test(**test_options)
 
 
+@pytest.mark.skipif(platform.system() == 'Windows', reason='Does not apply to Windows platform.')
```

Comment:
can the reason be more explicit why this does not work on windows

Reply:
@louis2889184 any insights here?